iaso/api/setup_account.py

```diff
@@ -29,13 +39,22 @@ class SetupAccountSerializer(serializers.Serializer):
     user_username = serializers.CharField(max_length=150, required=True)
     user_first_name = serializers.CharField(max_length=30, required=False)
     user_last_name = serializers.CharField(max_length=150, required=False)
-    password = serializers.CharField(required=True)
+    user_email = serializers.EmailField(required=False)
+    password = serializers.CharField(required=False)
     user_manual_path = serializers.CharField(required=False)
-    modules = serializers.JSONField(required=True, initial=["DEFAULT"])  # type: ignore
+    email_invitation = serializers.BooleanField(required=False, default=False)
+    language = serializers.ChoiceField(
+        choices=[("en", "English"), ("fr", "Français")],
```

Comment:
```suggestion
        choices=settings.LANGUAGES
```

We should probably rely on the configured languages here rather than hardcoding them.